net/http/authz/authorizer.go

```diff
@@ -16,19 +16,28 @@ import (
 
 var ErrNotAuthorized = errors.New("not authorized")
 
+var builtinGrants []*Grant // initialized in loopback_authz.go
+
 type Authorizer struct {
 	raftDB        *raft.Service
 	raftPrefix    string
 	policyByRoute map[string][]string
+	extraGrants   map[string][]*Grant
```

Comment:
The name doesn't have to be an essay; it shouldn't try to convey everything about how the field is going to be used. It has to be unique and evocative enough to remind you what it is when seen in context, and to make it easy to find its definition.

Specifically, in this case, what I'm going for is a way to distinguish this name from just `grants`, since, ordinarily, grants are loaded from the db. So `extraGrants` says that. But I'm specifically not trying to describe what data structure we're using here or really anything about how it's organized, because that's less germane (and more readily inferred from the code).

So:

```go
extraGrants map[string][]*Grant // by policy
```

or

```go
builtinGrants map[string][]*Grant // by policy
```

These both seem pretty good to me. (I slightly prefer "extra" but don't feel strongly about it.) Actual uses will look like `a.builtinGrants[policy]` so it'll be clear what the key is.

If it's not clear from the code what the meaning of the field is, I'm happy to write a comment explaining more about its purpose. (But I would rather write that explanation in a comment than inside the variable name.)